v3/add_column_BQ_ds/deploy/0.0.3/cd-deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+tb release create --semver ${VERSION}
+tb --semver ${VERSION} datasource rm bq_pypi --yes
+tb --semver ${VERSION} deploy --fixtures --populate --wait
+tb release preview --semver ${VERSION}
```

Comment:
I guess there will be another PR to backfill and sync the new bq_pypi Data Source in preview with the live one, right?